packages/types/index.ts

```diff
@@ -34,7 +34,7 @@ export interface TestCaseResult {
   score?: number;
   status: TestCaseResultStatus;
   iterations: TestCaseIterationResult[];
-  type?: undefined;
+  type?: undefined | "IOS_EXPERIMENTAL";
```

Comment:
[To Investigate / Not mandatory] Replace undefined with a more explicit "ANDROID" string type?